src/Algebra/Graph/Relation/InternalDerived.hs

```diff
@@ -12,18 +12,19 @@
 -- "Algebra.Graph.Relation.Symmetric", "Algebra.Graph.Relation.Transitive" and
 -- "Algebra.Graph.Relation.Preorder" instead.
 -----------------------------------------------------------------------------
+
 module Algebra.Graph.Relation.InternalDerived (
     -- * Implementation of derived binary relations
-    ReflexiveRelation (..), SymmetricRelation (..), TransitiveRelation (..),
+    ReflexiveRelation (..), TransitiveRelation (..),
     PreorderRelation (..)
   ) where
 
-
 import Control.DeepSeq (NFData (..))
 
 import Algebra.Graph.Class
-import Algebra.Graph.Relation (Relation, reflexiveClosure, symmetricClosure,
-                               transitiveClosure, closure)
+import Algebra.Graph.Relation.Internal hiding (empty, vertex, overlay, connect, referredToVertexSet, consistent)
+import qualified Algebra.Graph.Relation as R (reflexiveClosure, transitiveClosure, closure, 
```

Comment:
It looks like most of the changes in this module are accidental. I don't think you need to change imports or add `R.` qualifications, since the only real change is the removal of `SymmetricRelation` from this module. Is that correct? If yes, could you please undo all unrelated changes?

Reply:
I need some distinction because the Class module import is needed to recognize `Preorder`, `Graph`, `Reflexive`, etc... I'm trying to do it without the qualifier but I'm getting some weird errors `‘empty’ is not a (visible) method of class ‘Graph’`